daemon/network.go

```diff
@@ -318,8 +318,15 @@ func (daemon *Daemon) createNetwork(cfg *config.Config, create types.NetworkCrea
 		}
 	}
 
+	var enableIPv6 bool
+	if create.EnableIPv6 != nil {
+		enableIPv6 = *create.EnableIPv6
+	} else {
+		enableIPv6 = networkOptions[netlabel.EnableIPv6] == "true"
```

Comment:
Is there any validation elsewhere on the value? It would be really unfortunate for a typo such as `com.docker.network.enable_ipv6=ture` to be silently ignored. Also, [should we be more liberal in what we accept as valid true/false values?](https://pkg.go.dev/strconv#ParseBool)